addons/print_sale/wizard/print_document_partner_wizard.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+from openerp import api, fields, models, _
+from openerp.exceptions import Warning
+
+
+class print_document_partner_wizard(models.TransientModel):
```

Comment:
I know there was a god reason for this wizard to exist ... but maybe adding some docstrings could help my good ol' brain :p .